File: example_bots/java/NeverBluffBot.java
/**
 * A bot that never bluffs
 *
 * This bot will only raise if it has a better hand than the board
 * otherwise it will call
 */
public class NeverBluffBot  {
    public static final String BOT_NAME = "Never bluff bot";

    public static int act(Observable obs) throws Exception {
        HandType myHand = obs.getMyHandType();
        HandType boardHand = obs.getBoardHandType();

        if(myHand.getValue() > boardHand.getValue()){
            return obs.getMinRaise();
        } else {
            return 1;
        }
    }

}

File: example_bots/python/never_bluff_bot.py
from poker_game_runner.state import Observation
from poker_game_runner.utils import Range, HandType
"""
  This bot will only raise, if its hand is better than the cards on the table.
"""


class Bot:
    def get_name(self):
        return "never_bluff_bot"

    def act(self, obs: Observation):

        my_hand = obs.get_my_hand_type()
        board = obs.get_board_hand_type()

        if my_hand > board:
            return obs.get_min_raise()
        else:
            return 1
